net/Socket.cpp

```diff
@@ -1302,6 +1383,33 @@ LocalServerSocket::~LocalServerSocket()
 #  define LOG_CHUNK(X)
 #endif
 
+#endif // !MOBILEAPP
+
+std::string StreamSocket::toString(WSState t) noexcept
+{
+    if( WSState::WS == t )
```

Comment:
Not a fan of vertical space wasteage myself, it's a non-renewable resource; Ash is a huge fan for some reason =) either way - good to fit in with the surrounding code - which (I hope) is an "modern editors help you, but don't make your screen taller" mode ;-)

Reply:
[my personal clang-format](https://jausoft.com/cgit/jaulib.git/tree/.clang-format) looks different indeed, but not here to make enemies on my first weeks (yet) .. so I double the line count and clean my glasses ;-)

Reply:
Surely I'd put these on one line, @mmeeks! (This doesn't benefit from multiple lines, it suffers.)

`return WSState::WS == t ? "WS" : "HTTP";`

Although, I would call it `state` and return `const char*` (create `string` only when necessary, not by default).

Reply:
changed to const char*, thank you